src/picongpu/submit/taurus-tud/k80_profile.tpl

```diff
@@ -24,18 +24,18 @@ TBG_queue="gpu"
 TBG_mailSettings="ALL"
```

Comment:
will need extra line's [like this](https://github.com/ComputationalRadiationPhysics/picongpu/blob/9663ff002bbd66ffad2821ef45c48da476a5cad0/src/picongpu/submit/hypnos/picongpu.profile.example#L38-L40) in taurus' `picongpu.profile.example`:
```bash
# send me a mail on BEGIN, END, FAIL, REQUEUE or ALL
export MY_MAIL="someone@example.com"
export MY_MAILNOTIFY="ALL"
```